src/playback/qml/Audacity/Playback/components/PlaybackMeterCustomisePopup.qml

```diff
@@ -0,0 +1,195 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Muse.UiComponents 1.0
+import Muse.Ui 1.0
+
+StyledPopupView {
+    id: root
+
+    contentWidth: 292
+    contentHeight: 248
+
+    margins: 12
+
+    ColumnLayout {
+        anchors.fill: parent
+        spacing: 12
+
+        RowLayout {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 124
+            
+            spacing: 12
+
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+
+                spacing: 12
+
+                StyledTextLabel {
+                    Layout.fillWidth: true
+
+                    text: qsTr("Meter style")
+                    horizontalAlignment: Text.AlignLeft
+                }
+                
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+
+                    color: ui.theme.backgroundPrimaryColor
+                    border.width: 1
+                    border.color: ui.theme.strokeColor
+                    radius: 2
+
+                    Item {
+                        anchors.fill: parent
+                        anchors.margins: 12
+
+                        RadioButtonGroup {
+                            id: meterStyleGroup
+
+                            width: parent.width
+
+                            orientation: Qt.Vertical
+
+                            Column {
```

Comment:
I think, you can use `model + delegate` from RadioButtonGroup (see flatRadioButtonSample in GeneralComponentsGallery)